build.js

```diff
@@ -83,6 +91,21 @@ const readme = shades
     .join('\n')
     .replace(/^/, '|PREVIEW|NAME|HSL|HEX|\n|:---:|:---:|:---:|:---:|\n');
 
+const nativeColors = {
```

Comment:
should be renamed to just `colors`

Reply:
colors is already used so i named it `colorsExport`